projects/audit-logging.md

```diff
@@ -0,0 +1,68 @@
+# Audit Logging
+
+## Background and description
+
+Audit logging describes the capability of capturing audit-trail relevant events of a system to meet compliance requirements. Such events may originate from the infrastructure (e.g. a Kubernetes cluster) up to the application-level. It is a capability that is particularly relevant for providers of enterprise software.
+
+Unlike regular application logs, audit logs are usually subject to long retention periods and software providers must guarantee their completeness (i.e. guarantee of delivery).
+
+Examples of audit logs include:
+- permission changes (e.g. of a service account or application user)
+- modification of data
+- accessing sensitive information
+- failed login attempts
+
+### Current challenges
+
+Audit Logging is currently not within the scope of OpenTelemetry
+
+- no semantic conventions for audit logs in OTEL
+- OTEL APIs/SDKs do not provide feedback to the application level whether data (in particular logs) have been successfully delivered to a remote endpoint. To guarantee delivery, either the SDK has to give those guarantees, or provide feedback to the application so that it can take care of guaranteed delivery itself.
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification#project-naming

```suggestion
- OTel APIs/SDKs do not provide feedback to the application level whether data (in particular logs) have been successfully delivered to a remote endpoint. To guarantee delivery, either the SDK has to give those guarantees, or provide feedback to the application so that it can take care of guaranteed delivery itself.
```